components/dashboard/src/admin/UserDetail.tsx

```diff
@@ -0,0 +1,238 @@
+                <button className="secondary danger ml-3" disabled={activity} onClick={toggleBlockUser}>{user.blocked ? 'Unblock' : 'Block'} User</button>
+                <button className="danger ml-3" disabled={activity} onClick={deleteUser}>Delete User</button>
+            </div>
+            <div className="flex mt-6">
+                <div className="w-40">
+                    <img className="rounded-full h-28" alt={user.fullName} src={user.avatarUrl} />
+                </div>
+                <div className="flex flex-col w-full">
+                    <div className="flex w-full mt-6">
+                        <Property name="Sign Up Date">{moment(user.creationDate).format('MMM D, YYYY')}</Property>
+                        <Property name="Remaining Hours"
+                            action={
+                                accountStatement && {
+                                    label: 'View Account Statement',
+                                    onClick: () => setViewAccountStatement(true)
+                                }
+                            }
+                        >{accountStatement?.remainingHours ? accountStatement?.remainingHours.toString() : '---'}</Property>
+                        <Property
+                            name="Plan"
+                            action={accountStatement && {
+                                label: (isProfessionalOpenSource ? 'Disable' : 'Enable') + ' Professional OSS',
+                                onClick: () => {
+                                    getGitpodService().server.adminSetProfessionalOpenSource(user.id, !isProfessionalOpenSource);
+                                }
+                            }}
+                        >{accountStatement?.subscriptions ? accountStatement.subscriptions.filter(s => Subscription.isActive(s, new Date().toISOString())).map(s => Plans.getById(s.planId)?.name).join(', ') : '---'}</Property>
+                    </div>
+                    <div className="flex w-full mt-6">
+                        <Property name="Feature Flags"
+                            action={{
+                                label: 'Edit Feature Flags',
+                                onClick: () => {
+                                    setEditFeatureFlags(true);
+                                }
+                            }}
+                        >{user.featureFlags?.permanentWSFeatureFlags?.join(', ') || '---'}</Property>
+                        <Property name="Roles"
+                            action={{
+                                label: 'Edit Roles',
+                                onClick: () => {
+                                    setEditRoles(true);
+                                }
+                            }}
+                        >{user.rolesOrPermissions?.join(', ') || '---'}</Property>
+                        <Property name="Student">{isStudent === undefined ? '---' : (isStudent ? 'Enabled' : 'Disabled')}</Property>
```

Comment:
**question**: Tried enabling for a gmail address but didn't work. Expected?

Reply:
Yes, it is not enabled in dev-staging